include/mxnet/lib_api.h

```diff
@@ -571,6 +572,10 @@ class CustomOp {
     create_opstate = func;
     return *this;
   }
+  CustomOp& isSubgraphOp(bool state) {
```

Comment:
Sounds this function should return true or false. How about using `setXXX`?

Reply:
done! changed it to `setIsSubgraphOp()` and removed argument